### ISSUE TITLE
split term: vertebral, cardiac, renal, and limb defects syndrome into generic and type 1

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/mondo#1804]
Term to split: vertebral, cardiac, renal, and limb defects syndrome

Name for new term:

 - vertebral, cardiac, renal, and limb defects syndrome
 - vertebral, cardiac, renal, and limb defects syndrome 1 https://www.omim.org/entry/617660

List properties that should be moved to new term

consult docs (TBW) for ths split pattern

this is driven by a new PS, add this to generic

```
name: vertebral, cardiac, renal, and limb defects syndrome
xref: OMIMPS:617660 {source="MONDO:equivalentTo"} ! vertebral, cardiac, renal, and limb defects syndrome
```

**Comment by nicolevasilevsky** [comment]
I don't see any info in the editors docs about splitting terms.

Could you point me to it?

Let me know if there are any issues on the PR. :)

**Comment by cmungall** [comment]
apologies, I made a mistake specifying this. I saw we were lacking a PS and assumed the prototype pattern. In fact we hve the structure we need, we just need to add the PS xref to the parent:

```
id: MONDO:0020831
name: congenital vertebral-cardiac-renal anomalies syndrome
subset: ordo_malformation_syndrome
synonym: "VCRL" EXACT [MONDO:cjm]
xref: OMIM:617660 {source="Orphanet:521438"}
xref: OMIM:617661 {source="Orphanet:521438"}
xref: Orphanet:521438 {source="MONDO:equivalentTo"}
is_a: MONDO:0015161 {source="Orphanet:521438"} ! multiple congenital anomalies/dysmorphic syndrome without intellectual disability
is_a: MONDO:0015506 {source="Orphanet:521438"} ! rare syndrome with cardiac malformations
is_a: MONDO:0019589 {source="Orphanet:521438"} ! syndromic genetic deafness
is_a: MONDO:0019721 {source="Orphanet:521438"} ! syndromic renal or urinary tract malformation
is_a: MONDO:0043008 {source="Orphanet:521438"} ! genetic multiple congenital anomalies/dysmorphic syndrome without intellectual disability
relationship: has_modifier HP:0000007 {source="Orphanet:521438"} ! Autosomal recessive inheritance


[Term]
id: MONDO:0060554
name: vertebral, cardiac, renal, and limb defects syndrome 1
synonym: "3-hydroxyanthranilic acidemia" RELATED [OMIM:617660]
synonym: "congenital NAD deficiency Disorder 1" RELATED [OMIM:617660]
synonym: "VCRL1" RELATED [OMIM:617660]
synonym: "vertebral, cardiac, renal, and limb defects syndrome 1; VCRL1" RELATED [OMIM:617660]
xref: OMIM:617660 {source="MONDO:equivalentTo"}
xref: UMLS:C4540004 {source="MONDO:equivalentTo", source="OMIM:617660"}
is_a: MONDO:0003847 ! Mendelian disease
intersection_of: MONDO:0020831 ! congenital vertebral-cardiac-renal anomalies syndrome
intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/4796 ! HAAO

[Term]
id: MONDO:0060555
name: vertebral, cardiac, renal, and limb defects syndrome 2
synonym: "congenital NAD deficiency disorder 2" RELATED [OMIM:617661]
synonym: "kynureninase deficiency, complete" RELATED [OMIM:617661]
synonym: "VCRL2" RELATED [OMIM:617661]
synonym: "vertebral, cardiac, renal, and limb defects syndrome 2; VCRL2" RELATED [OMIM:617661]
xref: OMIM:617661 {source="MONDO:equivalentTo"}
xref: UMLS:C4540014 {source="MONDO:equivalentTo", source="OMIM:617661"}
intersection_of: MONDO:0020831 ! congenital vertebral-cardiac-renal anomalies syndrome
intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/6469 ! KYNU
```

**Comment by nicolevasilevsky** [comment]
this has been addressed, the OMIMPS xref'd on MONDO:0020831
